src/fixtures.ts

```diff
@@ -226,6 +226,19 @@ export class FixturePool {
   }
 }
 
+// We'll be using multiple pools.
+const currentFixturePool = new FixturePool();
+
+export async function setCurrentFixturePool(fixturePool: FixturePool) {
```

Comment:
I thought that every it / describe would actually have an explicit pointer to its tool, so there will be no 'current' pool.